batchupload/uploader.py

```diff
@@ -345,8 +356,14 @@ def main(*args):
         '(optional)\n'
         '\t-confirm Whether to output a confirmation after each upload '
         'attempt (optional)\n'
+        '\t-test Whether to do mock upload, simply outputting to commandline. '
+        '(optional)\n'
         '\t-nochunk Whether to turn of chunked uploading, this is slow '
-        'and does not support files > 100Mb (optional)\n'
+        'and does not support files > 100Mb (optional, type:FILES only)\n'
+        '\t-only:PATH to file containing list of urls to upload, skipping all '
+        'others. One entry per line. (optional, type:URL only)\n'
+        '\t-skip:PATH to file containing list of urls to skip, uploading all '
+        'others. One entry per line. (optional, type:URL only)\n'
```

Comment:
Since -only and -skip are mutually exclusive, maybe it should say so in the usage (and also in what order they're evaluated, even though ideally the user would never attempt to use both).